torchtitan/train.py

```diff
@@ -492,11 +492,13 @@ def forward_backward_step(
                 )
                 if self.pp_has_first_stage:
                     self.pp_schedule.step(
-                        inputs, target=targets, losses=losses, input_batch=inputs
+                        # TODO: input_batch kwarg only needed for CP, but
+                        # autoparallel doesn't accept kwargs in its forward
+                        inputs, target=targets, losses=losses #, input_batch=inputs
```

Comment:
Curious, why does CP need `input_batch`?

Reply:
oh, there was a change to remove the need of `input_batch`. We may want to do the same change to autoparalle.